package.json

```diff
@@ -122,7 +124,8 @@
     "tslint-config-prettier": "^1.14.0",
     "tslint-react": "^3.6.0",
     "typescript": "^3.5.1",
-    "typescript-eslint": "^0.0.1-alpha.0"
+    "typescript-eslint": "^0.0.1-alpha.0",
```

Comment:
Hmm. This seems to be a dead npm package. I don't see anywhere where it's used in our codebase. I see use of the `@typescript-eslint` monorepo of packages. Perhaps this was a typo @alex?

Reply:
Hmm. It very well could have been. I can open an issue to remove it.